README.md

```diff
@@ -68,25 +68,25 @@ This repo contains subgraphs defined using a set of standardized schemas. These
 | Protocol | Status | Versions † | Deployments |
 | ------- | :------: | --- | --- |
 | **DEX AMM** | | |
-| [Apeswap](https://apeswap.finance/) | ✅ | 1.3.0 / 1.1.3 / 1.0.0 | [![Apeswap BSC](./docs/images/chains/bsc.png)](https://thegraph.com/hosted-service/subgraph/messari/apeswap-bsc) [![Apeswap Polygon](./docs/images/chains/matic.png)](https://thegraph.com/hosted-service/subgraph/messari/apeswap-polygon) |
+| [Apeswap](https://apeswap.finance/) | 🛠 | 1.3.0 / 1.1.4 / 1.0.0 | [![Apeswap BSC](./docs/images/chains/bsc.png)](https://thegraph.com/hosted-service/subgraph/messari/apeswap-bsc) [![Apeswap Polygon](./docs/images/chains/matic.png)](https://thegraph.com/hosted-service/subgraph/messari/apeswap-polygon) |
```

Comment:
I think we can leave this as prod-ready, as the fix is minor

Reply:
Set to production ready.